examples/tutorials/atomic_contributions_tutorial_data/logBB.sdf

```diff
@@ -0,0 +1,15907 @@
+MolID_203
```

Comment:
Could we replace this with a smaller test file? We should only need one or a few molecules to test desiured functionality

Reply:
> 
> 
> Could we replace this with a smaller test file? We should only need one or a few molecules to test desiured functionality

We should have enogh data to build "acceptable quality" model, otherwise interpretation will return absolutely meaningless results. Tutorial was created to demonstrate the method's performance. I used as little data, as was enough to achieve ~70% accruacy